util/sqlexec/utils_test.go

```diff
@@ -0,0 +1,392 @@
+		},
+		{
+			name:   "discard extra arguments",
+			input:  "select * from 1",
+			params: []interface{}{4, 5, "rt"},
+			output: "select * from 1",
+			err:    "",
+		},
+		{
+			name:   "%? missing arguments",
+			input:  "select %? from %?",
+			params: []interface{}{4},
+			err:    "missing arguments.*",
+		},
+		{
+			name:   "nil",
+			input:  "select %?",
+			params: []interface{}{nil},
+			output: "select NULL",
+			err:    "",
+		},
+		{
+			name:   "int",
+			input:  "select %?",
+			params: []interface{}{int(3)},
+			output: "select 3",
+			err:    "",
+		},
+		{
+			name:   "int8",
+			input:  "select %?",
+			params: []interface{}{int8(4)},
+			output: "select 4",
+			err:    "",
+		},
+		{
+			name:   "int16",
+			input:  "select %?",
+			params: []interface{}{int16(5)},
+			output: "select 5",
+			err:    "",
+		},
+		{
+			name:   "int32",
+			input:  "select %?",
+			params: []interface{}{int32(6)},
+			output: "select 6",
+			err:    "",
+		},
+		{
+			name:   "int64",
+			input:  "select %?",
+			params: []interface{}{int64(7)},
+			output: "select 7",
+			err:    "",
+		},
+		{
+			name:   "uint",
+			input:  "select %?",
+			params: []interface{}{uint(8)},
+			output: "select 8",
+			err:    "",
+		},
+		{
+			name:   "uint8",
+			input:  "select %?",
+			params: []interface{}{uint8(9)},
+			output: "select 9",
+			err:    "",
+		},
+		{
+			name:   "uint16",
+			input:  "select %?",
+			params: []interface{}{uint16(10)},
+			output: "select 10",
+			err:    "",
+		},
+		{
+			name:   "uint32",
+			input:  "select %?",
+			params: []interface{}{uint32(11)},
+			output: "select 11",
+			err:    "",
+		},
+		{
+			name:   "uint64",
+			input:  "select %?",
+			params: []interface{}{uint64(12)},
+			output: "select 12",
+			err:    "",
+		},
+		{
+			name:   "float32",
+			input:  "select %?",
+			params: []interface{}{float32(0.13)},
+			output: "select 0.13",
+			err:    "",
+		},
+		{
+			name:   "float64",
+			input:  "select %?",
+			params: []interface{}{float64(0.14)},
+			output: "select 0.14",
+			err:    "",
+		},
+		{
+			name:   "bool on",
+			input:  "select %?",
+			params: []interface{}{true},
+			output: "select 1",
+			err:    "",
+		},
+		{
+			name:   "bool off",
+			input:  "select %?",
+			params: []interface{}{false},
+			output: "select 0",
+			err:    "",
+		},
+		{
+			name:   "time 0",
+			input:  "select %?",
+			params: []interface{}{time.Time{}},
+			output: "select '0000-00-00'",
+			err:    "",
+		},
+		{
+			name:   "time 1",
+			input:  "select %?",
+			params: []interface{}{time.Date(2019, 1, 1, 0, 0, 0, 0, time.UTC)},
+			output: "select '2019-01-01 00:00:00'",
+			err:    "",
+		},
+		{
+			name:   "time 2",
+			input:  "select %?",
+			params: []interface{}{time2},
+			output: "select '2018-01-23 04:03:05'",
+			err:    "",
+		},
+		{
+			name:   "time 3",
+			input:  "select %?",
+			params: []interface{}{time.Unix(0, 888888888)},
+			output: "select '1970-01-01 08:00:00.888888'",
```

Comment:
In my timezone this fails.
```suggestion
			params: []interface{}{time.Unix(0, 888888888).UTC()},
			output: "select '1970-01-01 00:00:00.888888'",
```

Reply:
Looks like I need to backport this commit..